module/VuFindSearch/src/VuFindSearch/Command/GetQueryBuilderCommand.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+/**
+ * Command to fetch a query builder object from a backend (if supported).
+ *
+ * PHP version 7
+ *
+ * Copyright (C) Villanova University 2021.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 2,
+ * as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+ *
+ * @category VuFind
+ * @package  Search
+ * @author   Demian Katz <demian.katz@villanova.edu>
+ * @license  http://opensource.org/licenses/gpl-2.0.php GNU General Public License
+ * @link     https://vufind.org
+ */
+namespace VuFindSearch\Command;
+
+use VuFindSearch\Backend\BackendInterface;
+
+/**
+ * Command to fetch a query builder object from a backend (if supported).
+ *
+ * @category VuFind
+ * @package  Search
+ * @author   Demian Katz <demian.katz@villanova.edu>
+ * @license  http://opensource.org/licenses/gpl-2.0.php GNU General Public License
+ * @link     https://vufind.org
+ */
+class GetQueryBuilderCommand extends AbstractBase
```

Comment:
I can't seem to find where this command is used apart from the tests? 

Do we want to expose the query builder if we are not exposing the connector?

Reply:
It is used in the DevToolsController; that's an odd special case because it is a development-oriented tool specifically designed to expose the internal behavior of the backend. (You can paste in a minified search query from VuFind's search table, and it will dump out a whole bunch of useful information about it, including the output of the query builder).